Sources/App/Core/SwiftVersion+Build.swift

```diff
@@ -19,15 +19,14 @@ extension SwiftVersion {
     static let v5_3: Self = .init(5, 3, 3)
     static let v5_4: Self = .init(5, 4, 0)
     static let v5_5: Self = .init(5, 5, 2)
-    static let v5_6: Self = .init(5, 6, 0)
+    static let v5_6: Self = .init(5, 6, 1)
 
     /// Currently supported swift versions for building
     static var allActive: [Self] {
         [.v5_3, .v5_4, .v5_5, .v5_6]
     }
 
     var xcodeVersion: String? {
-        // Match with https://gitlab.com/finestructure/swiftpackageindex-builder/-/blob/main/Sources/BuilderCore/SwiftVersion.swift#L41
```

Comment:
I believe this is no longer necessary. Certainly, this file reference is no longer valid and the builder project now pulls in SwiftVersion from this project as a dependency, so I believe this comment is no longer needed.

However, I noticed that the builder still references an old version of this project as a dependency that does not include Swift 5.6, which might need looking at.

Reply:
I haven't investigated this further but I'm pretty sure this still required but the reference is in the SPIManifest project not in builder anymore.

Reply:
Bumping the versions in the Server project is mostly irrelevant or for display only. For instance, the Xcode version is only shown on a build detail page. It's purely for display. We could even consider dropping the version details entirely or at least truncate it to x.y format to avoid having to deal with it.

The Swift version is only relevant in in its x.y part as well. We're actually triggering `5.6` when triggering a build and it's the builder who picks whatever latest Swift/Xcode version in that bracket it's got.

Here, we can't quite drop the patch version because our type has it. But it doesn't really matter what the patch version is. It gets saved to the `builds` table when we trigger but it's not actually even the true trigger version. I.e. if we didn't update the server and trigger 5.6.0 but the builder has 5.6.2 we'd save 5.6.0 in builds but the build was actually run with 5.6.2.

The true type should really be a Swift version without a patch field.

Reply:
I agree with your philosophy here but for as long as we're keeping patch versions in there they should be correct. Something to think about for a future change, though.

Reply:
Sure, I just mentioned it to clarify that this isn't operationally important or risky and we can deploy these kinds of changes freely on the server side.